assets/js/modules/adsense/dashboard/adsense-module-status.js

```diff
@@ -319,10 +319,11 @@ class AdSenseModuleStatus extends Component {
 						/>
 					) }
 
-					{ googlesitekit.canAdsRun && googlesitekit.modules.adsense.setupComplete && accountStatus && (
+					{ googlesitekit.canAdsRun && ( googlesitekit.modules.adsense.setupComplet || 'account-connected' === accountStatus ) && accountStatus && (
```

Comment:
Missing `e` in `setupComplete`
```suggestion
					{ googlesitekit.canAdsRun && ( googlesitekit.modules.adsense.setupComplete || 'account-connected' === accountStatus ) && accountStatus && (
```